src/modules/post-comments.js

```diff
@@ -0,0 +1,26 @@
+const APP_ID = 'DUanzoHMk8l8HLimHh6p';
+
+export const postComments = async (username, comment, itemId) => {
+  const myHeaders = new Headers();
+  myHeaders.append('Content-Type', 'application/json');
+  myHeaders.append('Authorization', 'Basic bWFqZDowMDAw');
+  console.log(itemId);
```

Comment:
[REQUIRED] Kindly remove all `console.log` from your code in order to make your project more professional